gcapi/gcapi.py

```diff
@@ -539,13 +537,47 @@ def __call__(
         else:
             return response
 
+    def run_external_job(
+        self, *, algorithm_name: str = None, inputs: Dict[str, any]
```

Comment:
For API consistency `algorithm_name` -> `algorithm` and take a slug (see upload_cases). Then, we can refactor `_get_algorithm` too.